kuttl-test-upgrade.yaml

```diff
@@ -14,13 +14,12 @@ kind: TestSuite
 crdDir: ./tests/_build/crds/
 artifactsDir: ./tests/_build/artifacts/
 kindContainers:
-  - local/opentelemetry-operator:e2e
   - ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator:v0.49.0
 commands:
-  - command: make cert-manager
+  - command: make undeploy # Undeploy the operator deployed during prepare-e2e
```

Comment:
this seems weird: 
* the caller should make sure the previous tests are cleaned
* is undeploy idempotent? Does it succeed if the operator was not previously deployed?